BSP/Simulator/Src/BSP_Timer.c

```diff
@@ -51,10 +48,13 @@ void BSP_Timer_Update()
 
 void BSP_Timer_Init(int Timer1Reload,int Timer2Reload,void *func1,void *func2)

```

Comment:
Now that I'm looking at this again, I would recommend using an enum approach to this. Very similar to how BSP_CAN does it (it also has 2 lines like this has 2 timers). So the Init function would take the enum of which timer, the reload for that timer, and the handler function. It would require you to call the Init function twice to initialize both timers, but that also allows us to initialize each timer in different places so they aren't tied to one another.